index.html

```diff
@@ -1104,6 +1104,68 @@ <h2>Extensibility</h2>
 
       </section>
 
+
+      <section>
+        <h2>Caching</h2>
+
+        <p>
+<a>Issuers</a> may wish to signal information related to caching a
+<a>credential</a> to <a>holders</a> and <a>verifiers</a>. Some issuers may
+assert that they do not want a credential to be cached, while others may
+specify certain caching rules and refresh time periods that are acceptable.
+        </p>
+
+        <p>
+Caching information for the credential in the Verifiable Credentials Data
+Model is provided by adding the following <a>properties</a>:
+        </p>
+
+        <dl>
+          <dt><var>cacheControl</var></dt>
+          <dd>
+The value of this property MUST be one or more of the acceptable values for
+the Cache-Control header in HTTP [[RFC7234]] Section
+<a href="https://tools.ietf.org/html/rfc7234#section-5.2">5.2. Cache-Control</a>.
+          </dd>
+          <dt><var>cacheRefreshService</var></dt>
```

Comment:
If we just change this to credentialRefreshService, then we could address #139 as well.

Reply:
Done.